modules/apps/site/site-insurance-site-initializer/remote-web-components/raylife-d2c-web/src/hooks/useStepWizard.js

```diff
@@ -14,6 +14,18 @@ export const useStepWizard = () => {
   useEffect(() => {
     _updateStepPercentage();
   }, [form]);
+  
+  const businessTotalFields = (properties) => {
+    let fieldCount = 4;
+    const bccAllowedToField = ["750", "1349"];
```

Comment:
I think in the future, we can't assign this value as fixed, right @joseabelenda ?

Maybe a way to workaround this, should make another request for categorization api?

Reply:
Is there a request that return the BCCs allowed for appear some fields? I was following the spreadsheet where some fields only appears when the product's BCC is 750 or 1349, for example.

Reply:
It's ok @jairmedeiros , I think we don't have this for now